scripts/bundleSize.sh

```diff
@@ -6,7 +6,7 @@
 # - The min value should be 5K less than the smallest service bundle and max should be 5K greater than the largest bundle
 # - Bundle sizes can be viewed by running a production build
 min=511
```

Comment:
```suggestion
min=520
```

Reply:
Howcome this PR makes the bundles soooo much bigger?

Reply:
I have no idea 🤷🏻‍♂️